guides/v2.2/magento-functional-testing-framework/release-2/getting-started.md

```diff
@@ -153,62 +153,66 @@ Example:
 http://127.0.0.1:4444/wd/hub
 ```
 
-### Step 5. [Optional] Copy `command.php` into Magento installation
+### Step 5. (Optional) Copy `command.php` into Magento installation {#add-cli-commands}
 
 If you are installing the MFTF not from your Magento installation, locate the `command.php` file in the MFTF:
 
-```
-magento2-functional-testing-framework/etc/config/command.php
-```
+`magento2-functional-testing-framework/etc/config/command.php`
 
 And copy it into your Magento installation under: 
 
-```
-magento2ce/dev/tests/acceptance/utils/command.php
-```
+`magento2/dev/tests/acceptance/utils/command.php`
 
 If you are installing the MFTF from inside your Magento installation, this is automatically done when you [build the project].
 
-{:.bs-callout .bs-callout-tip}
-If you do not have access to your Magento installation and cannot complete the above steps you will not be able to run tests using Magento CLI commands.
+{:.bs-callout .bs-callout-warning}
+If you do not have access to your Magento installation and cannot complete the step you will not be able to use the Magento CLI commands.
 
-### Step 6. Generate existing tests
+### Step 6. Enable the Magento CLI commands
```

Comment:
I had both of these as part of step 5 because if you can't complete step 5, you can't complete step 6 either.
Step 5 copies a file into the folder, and Step 6 allows MFTF to use that file (So if 5 is optional, 6 also has to be)

Reply:
Do I still need to allow the MFTF to use that file if my case doesn't require the optional step.
Your note says that it must be done after building the project (Step 3).
See my comment:  https://github.com/magento/devdocs/pull/2690/commits/4f2fd775f485c62ea4a2d197d2965b92c9e06255#r207984996

Reply:
I see what you mean. Yes, the note added says it's done if you `build:project`, so I see what you mean.
In that case, it's good.